doc/computation.rst

```diff
@@ -254,52 +259,52 @@ support weighted ``sum`` and weighted ``mean``.
 
 .. ipython:: python
 
-  coords = dict(month=('month', [1, 2, 3]))
+    coords = dict(month=("month", [1, 2, 3]))
 
-  prec = xr.DataArray([1.1, 1.0, 0.9], dims=('month', ), coords=coords)
-  weights = xr.DataArray([31, 28, 31], dims=('month', ), coords=coords)
+    prec = xr.DataArray([1.1, 1.0, 0.9], dims=("month",), coords=coords)
+    weights = xr.DataArray([31, 28, 31], dims=("month",), coords=coords)
 
 Create a weighted object:
 
 .. ipython:: python
 
-  weighted_prec = prec.weighted(weights)
-  weighted_prec
+    weighted_prec = prec.weighted(weights)
+    weighted_prec
 
 Calculate the weighted sum:
 
 .. ipython:: python
 
-  weighted_prec.sum()
+    weighted_prec.sum()
```

Comment:
The RST reference (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html) dictates a 3-spaces indentation for multi-line .. blocks. However I appreciate the frustration in not being to use the Tab key. Was it a conscious decision?

Reply:
no, not really. I was using 4 spaces since that's the indent in the python code. Actually, the tool only makes sure there is a consistent indentation depth (it enforces the indentation of the code).

If you look at the reference, though, the 3-spaces indentation is only implicitly mentioned in the syntax diagram:
```
+-------+-------------------------------+
| ".. " | directive type "::" directive |
+-------+ block                         |
        |                               |
        +-------------------------------+
```
but not in the text, it only says:
> The directive block [...] consists of any text on the first line of the directive after the directive marker, and any subsequent indented text.

So I'd assume 3 spaces is not mandatory.